### ISSUE TITLE
 The "nullable" column option is not supported, setting it is deprecated; No alternative

### ISSUE THREAD

**Issue by JoppeDC** [doctrine/annotations#275]
Not sure if this is the correct repo to post this issue. Let me know if its not, or if you need any more info.

The last few weeks, i've started getting deprecation errors using doctrine. The error seems to be thrown by Doctrine/dbal, but seems related to this bundle.

I've been looking for an alternative, but can't seem to find it. The `Column` in `doctrine/dbal` has the `notNull` option.

using:
```
doctrine/annotations                v1.6.0 
doctrine/cache                      v1.8.0 
doctrine/collections                v1.5.0 
doctrine/common                     v2.10.0
doctrine/data-fixtures              v1.3.1 
doctrine/dbal                       v2.9.2 
doctrine/doctrine-bundle            1.10.2 
doctrine/doctrine-cache-bundle      1.3.5  
doctrine/doctrine-fixtures-bundle   v2.4.1 
doctrine/doctrine-migrations-bundle 1.1.1  
doctrine/event-manager              v1.0.0 
doctrine/inflector                  v1.3.0 
doctrine/instantiator               1.2.0  
doctrine/lexer                      v1.0.1 
doctrine/migrations                 v1.8.1 
doctrine/orm                        v2.6.3 
doctrine/persistence                v1.1.0 
doctrine/reflection                 v1.0.0 
```

**Comment by alcaeus** [comment]
The deprecation you refer to was introduced in https://github.com/doctrine/dbal/pull/2846 which was first released as 2.7.0. The error is probably caused by ORM applying its own `nullable` column option from their annotations to the DBAL column specification. This is best reported in the ORM project if you are using ORM.

The only thing I can definitely say is that this deprecation is not caused by the annotation library itself - closing here.

**Comment by JoppeDC** [comment]
Thanks @alcaeus. Opened issue on the `doctrine/orm` repo.
